inc/ZipProvider.php

```diff
@@ -149,7 +149,7 @@ public function generate_zip_file() : bool {
 
 		$temp_zip_file = wp_tempnam( $this->get_zip_filename() );
 
-		if ( $zip->open( $temp_zip_file, ZipArchive::CREATE ) === true ) {
+		if ( $zip->open( $temp_zip_file, ZipArchive::OVERWRITE ) === true ) {
```

Comment:
Wondering if we should use `ZIPARCHIVE::CREATE | ZIPARCHIVE::OVERWRITE`. Would that also fix your issue?

Reply:
Yes, seems to work!